code/game/machinery/rechargestation.dm

```diff
@@ -152,6 +178,7 @@
 		else if(ismanipulator(P))
 			man_rating += P.rating
 	cell = locate(/obj/item/weapon/cell) in component_parts
+	beaker = locate(/obj/item/weapon/reagent_containers/glass/beaker) in component_parts

```

Comment:
how does this even work? It is getting just the type itself, not a reference to an object